howso/utilities/feature_attributes/relational.py

```diff
@@ -943,6 +945,12 @@ def _infer_feature_bounds(self,  # noqa: C901
                 if format_dt is not None:
                     min_v = epoch_to_date(min_v, format_dt, min_date_tz)
                     max_v = epoch_to_date(max_v, format_dt, max_date_tz)
+
+                if is_float_dtype(min_v):
+                    min_v = float(min_v)
+                if is_float_dtype(max_v):
+                    max_v = float(max_v)
```

Comment:
Wouldn't the observed values also need to be converted?

Reply:
Also, you are passing in the value of min/max, but this expects the dataframe column dtype?